diff --git a/constants.py b/constants.py
index 6f33ec3..4a54906 100644
--- a/constants.py
+++ b/constants.py
@@ -32,7 +32,7 @@ COL = 3
 KEYMAP = ['1', '2', '3', 'q', 'w', 'e', 'a', 's', 'd', 'z', 'x', 'c', '8',
           '9', '0']
 
-CARD_STYLES = ['pattern', 'number', 'word', 'custom']
+CARD_STYLES = ['pattern', 'number']
 NUMBER_STYLES_O = ['roman', 'product', 'chinese', 'word', 'mayan', 'incan']
 NUMBER_STYLES_C = ['lines', 'dots', 'star', 'hash', 'dice']
 
